src/app/components/user-profile-form/user-profile-form.component.html

```diff
@@ -35,22 +34,27 @@
     <textarea matInput type="text" formControlName="description"></textarea>
   </mat-form-field>
 
-  <button
-    mat-raised-button
-    color="primary"
-    style="color: var(--text-color)"
-    type="submit"
-    [disabled]="!userProfileForm.valid"
-  >
-    Submit
-  </button>
-  <button
-    mat-raised-button
-    color="accent"
-    style="color: var(--text-color)"
-    type="button"
-    (click)="onCancel()"
-  >
-    Cancel
-  </button>
+  <div style="margin-bottom: 1rem">
```

Comment:
same, and same on line 42, 52 . You could search in all .html files for 'style' and delete everywhere